web_client/stylesheets/uploadWidget.styl

```diff
@@ -0,0 +1,5 @@
+.modal-content
```

Comment:
Did you intend for these rules to be applied to every modal, or just the upload widget?

Reply:
Global -- it has already been upstreamed into Girder, so I can remove it here.